.dockerignore

```diff
@@ -4,7 +4,8 @@
 
 Dockerfile
 .dockerignore
-node_modules/
+**/node_modules/
+.git
```

Comment:
I guess this should have a trailing slash to make the entry only apply to directories.

Reply:
.dockerignore differs from .gitignore in that `.git` only targets the root level, it's not recursive without the wildcard, so it will always target only the git directory in the root.

Reply:
In theory it would match a file `.git` in root, but as we maintain the project with Git, it will always have a `.git` directory in root, and most operating systems don't allow files and directories to have the same path, so in practice this will never match a file with a path of `.git`.

Reply:
I agree. Would you mind to still make it read `.git/` for consistency with all other entries which also have a trailing slash if a directory is supposed to be matched?